### PR TITLE
problem in self register

### PR DESCRIPTION
**Pull request by arshiaor**
A Problem in self registering for members and member security access that need to solve.

### CODE REVIEW

File: src/Member.java [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
-        return super.toString()+"Member{" +
+        return super.toString() + "Member{" +
                 "memberId=" + memberId +
                 '}';
     }
 }
+
+class Member_CRUD{
+    Member[] members = new Member[300];
+    public void AddMember(String name, String family, int age, int nationalId, int memberId) {
```
Comment (amir8602) [inline src/Member.java:29]: AddMember ----> addMember 
camel case

**Comment by arshiaor** [comment]
Problem Solved!

File: src/Member.java (hunk 1 above)
Comment (arshiaor) [reply src/Member.java:29]: Yes,thanks

**Comment by amir8602** [comment]
why closed it???

**Comment by amir8602** [comment]
[Activation Code.txt](https://github.com/JavaTeam2021/Library/files/6468459/Activation.Code.txt)

**Comment by amir8602** [comment]
download

**Pull request closed without merging**